rules/integrations/google_workspace/collection_google_workspace_custom_gmail_route_created_or_modified.toml

```diff
@@ -0,0 +1,73 @@
+[metadata]
+creation_date = "2022/09/13"
+integration = "google_workspace"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/09/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Gmail is by default the mail routing service in Google Workspace. By default, all email from current Google Workspace
+users with accounts are routed through a domain's mail server for inbound and outbound mail. Adversaries can add a
+custom e-mail route for outbound mail to route these e-mails to their own inbox of choice for data gathering. This
+allows adversaries to capture sensitive information from e-mail and potential attachments, such as invoices or payment
+documents.
+"""
+false_positives = [
+    """
+    Administrators may create custom email routes in Google Workspace based on organizational policies, administrative
+    preference or for security purposes regarding spam.
+    """,
+]
+from = "now-130m"
+index = ["filebeat-*", "logs-google_workspace*"]
+interval = "10m"
```

Comment:
This seems like excessive overlap to look back 130m every 10m. If the lookback must be so wide, can you update the interval to run every 120m or similar